.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "dependencies/chisel"]
 	path = dependencies/chisel
 	url = https://github.com/chipsalliance/chisel.git
+[submodule "dependencies/diplomacy"]
+	path = dependencies/diplomacy
+	url = https://github.com/lordspacehog/diplomacy.git
```

Comment:
should be chipsalliance/diplomacy in the future.

Reply:
yeah, My current plan is to do a 2nd PR to cut it over,. or we can wait until diplomacy is published to ivy and i can just the setup over to use ivy.

Reply:
updated this PR to point at the new branch in the chipsalliance/diplomacy repo